src/configs/getAndroidConfig.ts

```diff
@@ -40,15 +46,72 @@ export default function getAndroidConfig(theme: Theme): Platform {
         destination: `${_.upperFirst(_.camelCase(theme))}DesignTokens.kt`,
         className: _.upperFirst(_.camelCase(theme)) + "DesignTokens",
         packageName: "io.element.android.compound",
+        filter: function(token: TransformedToken) {
+          return !isFontType(token) && isCoreColor.matcher(token);
+        },
         options: {
           showFileHeader: false,
+          outputReferences: true,
           import: [
             "androidx.compose.ui.graphics.Color",
             "androidx.compose.ui.unit.*",
             "androidx.compose.ui.text.*",
           ],
         },
       },
+      // If we find a way to describe semantic colors, it might be possible to move this to 'common'
```

Comment:
I couldn't put this directly in 'common' because it references tokens that are only available in the `themed` dictionary. There might be some combination of adding those resources and filtering those tokens out to get this working though.